src/Pdf.php

```diff
@@ -125,6 +125,31 @@ public function saveImage($pathToImage)
     }
 
     /**
+     * Save the file as images to the given path.
+     *
+     * @param string $storeDirectory
+     * @param string $file_name     
+     *
+     * @return array $files the full path to the newly created images.
+     */
+    public function extractAllPagesAsImages($storeDirectory, $file_name)
+    {
+        $pages = $this->getNumberOfPages();
```

Comment:
This variable does not contain the pages itself, but the number of page. Best to name it `$numberOfPages`